ostd/src/console.rs

```diff
@@ -5,23 +5,38 @@
 use core::fmt::Arguments;
 
 /// Prints formatted arguments to the console.
-pub fn print(args: Arguments) {
-    crate::arch::console::print(args);
+///
+/// The message printed by this function will exclusively be printed without
+/// being mixed with other messages. However two distinct executions of this
+/// function on the same core may not be contiguous.
```

Comment:
The verb "mixed" should be replaced with "interleaved" to be more accurate. And the word "exclusively" should be done with "atomically".

```rust
/// However two distinct executions of this
/// function on the same core may not be contiguous.
```

The latter half of the Rust doc looks too... obvious. Of course, on a SMP machine, the calls to any single function may be concurrent and their side effects (like generated messages) may interleave with each other. Reading the comments raises more surprise (why explaining the most obvious?), than giving help. I suggest removing it.